ZAPD/ZCollision.cpp

```diff
@@ -291,46 +302,64 @@ CameraDataList::CameraDataList(ZFile* parent, const std::string& prefix,
 	assert(numElements < 10000);
 
 	offset_t cameraPosDataSeg = rawDataIndex;
+	uint32_t numDataTotal;
+	uint32_t cameraPosDataOffset2;
+	uint32_t cameraPosDataSegEnd = rawDataIndex;
+	bool isSharpOcarina = false;
+
```

Comment:
A comment somewhere explaining that with Sharp Ocarina the camera position data comes after the camera data entries while for vanilla the position data comes before would be good.